### PR TITLE
fix: booking page: location icon is not showing up (CALCOM-9951)

### LINKED ISSUE #9951: [CAL-2104] booking page: location icon is not showing up
we used to show the icon of the app next to the name.

now we only show the generic location icon next to "Discord"

[Source image](https://github.com/calcom/cal.com/assets/8019099/9a228119-6ea1-443f-b472-145eae84d050)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2104](https://linear.app/calcom/issue/CAL-2104/booking-page-location-icon-is-not-showing-up)</sub>

### PR DESCRIPTION
**Pull request by gitstart-calcom**
Fixes #9951 

### Demo

Light Mode

https://jam.dev/c/8d0a75fd-2765-4782-9ee5-e30fc88a2f0f

251741740-6cd691c9-23b4-4088-a34b-319329d59327

Dark Mode

https://jam.dev/c/30650773-d836-4254-8e3f-ec682eeea57a

251741871-b348af85-9dce-470e-a6c2-7b385f170bbd

image

### CODE REVIEW

**Comment by Jaibles** [comment]
Looks good!

**Comment by PeerRich** [comment]
make sure to fix this in the new booker, not the old one. you find the new booker in /atoms/

**Review by PeerRich — DISMISSED** [review]

File: apps/web/components/booking/AvailableEventLocations.tsx [hunk 1]
```diff
@@ -28,14 +46,7 @@ export function AvailableEventLocations({ locations }: { locations: Props["event
...
         ) : (
-          <img
-            src={eventLocationType.iconUrl}
-            className={classNames(
-              "ml-[2px] h-4 w-4 opacity-70 ltr:mr-[10px] rtl:ml-[10px] dark:opacity-100 ",
-              !eventLocationType.iconUrl?.startsWith("/app-store") ? "dark:invert-[.65]" : ""
-            )}
-            alt={`${eventLocationType.label} icon`}
-          />
+          renderIcon(eventLocationType, false)
```
Comment (PeerRich) [inline apps/web/components/booking/AvailableEventLocations.tsx:49]: why is this not a react component?

File: apps/web/components/booking/AvailableEventLocations.tsx [hunk 2]
```diff
@@ -45,7 +56,43 @@ export function AvailableEventLocations({ locations }: { locations: Props["event
...
+          return (
+            <div key={`${location.type}-${index}`} className="flex flex-row items-center text-sm font-medium">
+              {renderIcon(eventLocationType, true)}
+              <p className="line-clamp-1">{translatedLocation}</p>
+            </div>
+          );
+        })}
+      </div>
+    );
+  };
```
Comment (sean-brydon) [inline apps/web/components/booking/AvailableEventLocations.tsx:78]: Does this need to be a function? Can't we just return this as a component also?

**Comment by gitstart-calcom** [comment]
@PeerRich @sean-brydon thanks for the review! we are going to work on it

**Review by PeerRich — DISMISSED** [review]

**Comment by PeerRich** [comment]
approved but @sean-brydon can you reveiw this too?

**Review by sean-brydon — DISMISSED** [review]
Code looks great - we don't appear to be using the component tho? 

Please add this component to be rendered in L:132 cal.com/packages/features/bookings/components/event-meta/Details.tsx 

We can probably remove the component this will be replacing too

**Comment by gitstart-calcom** [comment]
Alright @sean-brydon, thanks for the feedback 😄

**Review by sean-brydon — APPROVED** [review]
Great! Thanks for making them changes! 

LGTM :)

**Pull request merged**